File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.size() == 0) {return 0;}
        int k = 0, i;
        for (i = 1; i < nums.size(); i++) {
            if (nums[k] != nums[i]) {
                k++;
                nums[k] = nums[i];
            }
        }
        return k + 1;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        if len(nums) == 1:
            return 1
        ans, j = 1, 1
        for i in range(len(nums)-1):
            while (nums[i] == nums[j] and j < len(nums)-1):
                j += 1
            if nums[i] != nums[j]:
                ans += 1
            nums[i+1] = nums[j]
        return ans
        